pkgs/applications/networking/instant-messengers/zoom-us/default.nix

```
@@ -168,6 +168,10 @@ stdenv.mkDerivation rec {
     makeWrapper $out/opt/zoom/.zoom $out/opt/zoom/zoom \
       --prefix LD_LIBRARY_PATH ":" ${libs}
 
+    mv $out/opt/zoom/ZoomWebviewHost $out/opt/zoom/.ZoomWebviewHost
+    makeWrapper $out/opt/zoom/.ZoomWebviewHost $out/opt/zoom/ZoomWebviewHost \
```

```suggestion
    wrapProgram $out/opt/zoom/ZoomWebviewHost \
```

I also needed to add `libcef`, `libffmpeg` (used the one from vivaldi-codecs), `libsForQt5.qt5.qtbase` to it. But it still failed with `undefined symbol: cef_get_exit_code`